doc/news/_preparation_next_release.md

```diff
@@ -25,15 +25,20 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.9.
 
 ## Highlights
 
-- <<HIGHLIGHT1>>
-- <<HIGHLIGHT2>>
-- <<HIGHLIGHT3>>
+- TOML plugin
+- <<HIGHLIGHT>>
+- Cleanup
 
-### <<HIGHLIGHT1>>
+### TOML
 
-### <<HIGHLIGHT2>>
+- Added the TOML plugin, which can read and write TOML files using flex and bison. _(Jakob Fischer)_
```

Comment:
Are all the TOML entries in the news doc a rebase/merge error?

Reply:
Mh, yeah I may have re-introduced that during rebase. I'll clean that up.

Reply:
Thank you very much :sparkle: